src/main/java/seedu/ezmanager/storage/Storage.java

```diff
@@ -205,6 +218,7 @@ public Storage(String filePath) {
 
             return members;
         } catch (FileNotFoundException | NoSuchElementException e) {
+            EzLogger.log(Level.INFO, "Creating file for storage...");
```

Comment:
Hi Sam I think this should be EZLogger Level.warning as it is in exception clause.